db/index.js

```diff
@@ -0,0 +1,23 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  database: 'cava_table',
```

Comment:
Super nitpicky, but you should take out the last comma on this line since it's the last key/value pair in this object